Stronger/Stronger.swift

```diff
@@ -21,9 +21,9 @@ struct Stronger: App {
         WindowGroup {
             ZStack {
                 if completedOnboardingFlow {
-                    HomeReal()
+                    HomeView()
                 } else {
-                    EmptyView()
+                    Welcome()
```

Comment:
I think this change is causing the crash. The `Welcome` view is expecting to be rendered in an onboarding stack and pulls the respective environment objects from the SwiftUI environment: https://swiftpackageindex.com/stanfordspezi/spezionboarding/1.1.0/documentation/spezionboarding/onboardingstack